tests/features/environment.py

```diff
@@ -122,7 +122,7 @@ def before_all(context):
 def show_env_changes(env_old, env_new):
     """Print out all test-specific env values."""
     print("--- os.environ changed values: ---")
-    all_keys = set(list(env_old.keys()) + list(env_new.keys()))
+    all_keys = set(env_old) | set(env_new)
```

Comment:
Maybe cleaner to just use `env_old.keys() | env_new.keys()`?

Reply:
:+1: